monai/networks/nets/efficientnet.py

```diff
@@ -161,9 +161,9 @@ def __init__(
             self._se_adaptpool = adaptivepool_type(1)
             num_squeezed_channels = max(1, int(in_channels * self.se_ratio))
             self._se_reduce = conv_type(in_channels=oup, out_channels=num_squeezed_channels, kernel_size=1)
-            self._se_reduce_padding = _make_same_padder(self._se_reduce, [1, 1])
+            self._se_reduce_padding = _make_same_padder(self._se_reduce, [1, 1, 1])
```

Comment:
Thanks for posting this issue, but this change is still hard coded. suggest to use:
```
[1] * spatial_dims
```

Reply:
Agreed, that's better. Will do.